src/main/scala/com/gu/contentapi/services/Ophan.scala

```diff
@@ -0,0 +1,39 @@
+package com.gu.contentapi.services
+
+import okhttp3.{ OkHttpClient, Request }
+import com.gu.contentapi.models.Event
+import com.gu.contentapi.utils.{ Encoding }
+
+object Ophan {
+
+  private val client = new OkHttpClient()
+  private val OphanUrl = "https://ophan.theguardian.com/i.gif"
+
+  //TODO handle time when the event has been received
+
+  def send(events: Seq[Event]): Unit = {
+    events foreach { e =>
+      Thread.sleep(270000 / events.length) /* send events over a period of 4 minutes and 30 seconds */
+      Ophan.send(e)
+    }
+
+    println(s"Events sent to Ophan: ${events.length}")
```

Comment:
It's actually the only truly useful debug line to have to get a sense of how many events we're sending to Ophan